src/app/projects/projects-table/projects-table.component.html

```diff
@@ -1,15 +1,13 @@
-<app-menubar></app-menubar>
-
-<div class="ui-g" id="projects-table">
-  <div class="ui-g-9">
+  <div class="ui-g" id="projects-table">
+  <div class="ui-g-10">
     <h5>Dostępne projekty</h5>
     <p-table [value]="projects">
       <ng-template pTemplate="header">
         <tr>
-          <th>Lp</th>
+          <th>Id</th>
```

Comment:
Nie widzę sensu tej kolumny (id nie muszą być po kolei i mogą to być jakieś randomowo wyglądające, duże liczby)